narwhal/benchmark/benchmark/utils.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, message, error):
 class PathMaker:
     @staticmethod
     def binary_path():
-        return join('..', 'target', 'release')
+        return join('../..', 'target', 'release')
```

Comment:
This is fixing a bug since the repository move. The created symlinks pointing to the wrong folder (now `target` lives one level above - on the root of SUI repo).